src/GusApi/Context/Context.php

```diff
@@ -18,7 +18,7 @@ public function __construct()
 
     public function setOptions(array $options): bool
     {
-        return stream_context_set_option($this->context, $options);
+        return stream_context_set_options($this->context, $options);
```

Comment:
Composer php version is `"php" : "^8.0",` meaning
we need to add check `PHP_VERSION_ID >=  80400` otherwise we have to use stream_context_set_option

Reply:
@johnzuk I've pushed another version - I hope PHPStan should be happy with the latest version.